### PR TITLE
[HOT-FIX] Fix the test case failures in DDLSuite

### PR DESCRIPTION
**Pull request by gatorsmile**
#### What changes were proposed in this pull request?

Test cases failure in `DDLSuite` introduced in https://github.com/apache/spark/pull/13427 is causing build failure. This PR is to fix it.
#### How was this patch tested?

Local testing.

### CODE REVIEW

**Comment by gatorsmile** [comment]
@rxin @andrewor14 @srowen It causes a build break: https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/59698/consoleFull

Please merge it ASAP. Thanks!

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala [hunk 1]
```diff
@@ -310,15 +310,13 @@ trait CheckAnalysis extends PredicateHelper {
             failAnalysis(
               s"""
                  |Hive support is required to select over the following tables:
-                 |${s.catalogTable.identifier}
-               """.stripMargin)
+                 | ${s.catalogTable.identifier}""".stripMargin)
```
Comment (viirya) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala]: Why this change?

Comment (gatorsmile) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala]: Message formatting issues.

**Comment by gatorsmile** [comment]
Since this has been resolved by @andrewor14 , let me close it.

**Comment by JackyPJB** [comment]
您好，感谢您的岗位推荐。我对这个职位不是很有兴趣，如果有互联网企业的大数据岗位，请再推荐给我。


________________________________
发件人: Xiao Li <notifications@github.com>
发送时间: 2016年6月1日 2:28
收件人: apache/spark
主题: [apache/spark] [HOT-FIX] Fix the test case failures in DDLSuite (#13430)

What changes were proposed in this pull request?

Test cases failure in DDLSuite introduced in #13427<https://github.com/apache/spark/pull/13427>

How was this patch tested?

Local testing.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/apache/spark/pull/13430

Commit Summary

  *   hot fix.

File Changes

  *   M sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala<https://github.com/apache/spark/pull/13430/files#diff-0> (6)
  *   M sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala<https://github.com/apache/spark/pull/13430/files#diff-1> (8)

Patch Links:

  *   https://github.com/apache/spark/pull/13430.patch
  *   https://github.com/apache/spark/pull/13430.diff

―
